package exception

class MyException: Exception {
 internal constructor(): super()

 internal constructor(cause: Throwable?): super(cause)
}
